README.md

```diff
@@ -1,7 +1,71 @@
 # HTML Validation
+## Problem Statement 
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/nYglnxMUixM?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
+By now, you've probably noticed that the nested structure of HTML elements lends itself to multiple typos, as well as the opportunity to wonder if you've actually closed all of your tags or not. While text editors can help, and modern day browsers are pretty good at compensating for malformed HTML, neither can always accurately predict a developer's intentions. Additionally, what Chrome chooses to display when an HTML error occurs could be very different from what Safari, Firefox, Internet Explorer, or any other browser chooses to display. How can we be certain that the HTML that we are writing is both within the current HTML standards, and error-free? Luckily, there are validation services that can help.
```

Comment:
Quick point. I recommend, but haven't gotten militant about hard wrapping the text. The reason is I'd like to comment "whether vs. if" but I can't target _that_ line. Teach your editor that hard-wrap trick please. Especially for new text, if it's wrapped, it starts the precedent.